rootfs/scheduler/__init__.py

```diff
@@ -427,6 +427,9 @@ def _update_application_service(self, namespace, name, app_type, image, routable
             # Find if target port exists already, update / create as required
             if routable:
                 port = self._get_port(image)
+                if port is None:
```

Comment:
i created a issue https://github.com/deis/controller/issues/656 to discuss what should be the behaviour